instrumentation/rocketmq-client-4.8/library/src/main/java/io/opentelemetry/instrumentation/rocketmq/RocketMqConsumerInstrumenter.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.rocketmq;
+
+import io.opentelemetry.context.Context;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+import java.util.List;
+import org.apache.rocketmq.common.message.MessageExt;
+
+final class RocketMqConsumerInstrumenter {
+
+  private final Instrumenter<MessageExt, MessageExt> singleProcessInstrumenter;
+  private final Instrumenter<MessageExt, MessageExt> batchProcessInstrumenter;
+  private final Instrumenter<Void, Void> receiveInstrumenter;
+
+  RocketMqConsumerInstrumenter(
+      Instrumenter<MessageExt, MessageExt> singleProcessInstrumenter,
+      Instrumenter<MessageExt, MessageExt> batchProcessInstrumenter,
+      Instrumenter<Void, Void> receiveInstrumenter) {
+    this.singleProcessInstrumenter = singleProcessInstrumenter;
+    this.batchProcessInstrumenter = batchProcessInstrumenter;
+    this.receiveInstrumenter = receiveInstrumenter;
+  }
+
+  Context startSpan(Context parentContext, List<MessageExt> msgs) {
```

Comment:
WDYT about renaming this method to `start()` for more symmetry with `Instrumenter`?
Also, should we add a `shouldStart()` method too?